zfit/models/basic.py

```diff
@@ -240,3 +241,98 @@ class ExponentialPDFRepr(BasePDFRepr):
     hs3_type: Literal["Exponential"] = Field("Exponential", alias="type")
     x: SpaceRepr
     lam: Serializer.types.ParamTypeDiscriminated
+
+
+class Voigt(BasePDF, SerializableMixin):
+    _N_OBS = 1
+
+    def __init__(
+        self,
+        m: ztyping.ObsTypeInput,
+        sigma: ztyping.ParamTypeInput,
+        gamma: ztyping.ParamTypeInput,
+        obs: ztyping.ObsTypeInput = None,
+        *,
+        extended: ExtendedInputType = None,
+        norm: NormInputType = None,
+        name: str = "Voigt",
+    ):
+        """Voigt profile.
+
+        The Voigt profile is a convolution of a 1-D Gaussian distribution with standard deviation sigma and a 1-D Cauchy distribution with half-width at half-maximum gamma.
+
+        The (unnormalized) Voigt profile shape is defined as
+
+        .. math::
+            V(x; \\sigma, \\gamma) = Re[w(z)]
```

Comment:
```suggestion
        .. math::
        
            V(x; \\sigma, \\gamma) = Re[w(z)]
```
needs a space afaik (rest docs: https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-math)